pkg/controllers/operator/v2/endpoint/types.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 // podEndpoint represents a Pod/CiliumEndpoint
-type podEndpoint struct {
+type PodEndpoint struct {
```

Comment:
do we need to change this to an exposed struct (capitalized)?

Reply:
reverted